src/OpenTelemetry.Extensions.Enrichment/EnrichmentExtensions.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="EnrichmentExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Extensions.Enrichment;
+
+public static class EnrichmentExtensions
```

Comment:
nit: Rename to something a little less generic maybe `TracerProviderBuilderEnrichmentExtensions`. Just to potentially eliminate future collisions in case this project gets split up. IIRC doc generation has a problem if the same class name is found in multiple projects. If the intention is to have metrics and logs (eventually) in the same class, maybe `OpenTelemetryProviderBuilderEnrichmentExtensions`?

Reply:
Renamed to OpenTelemetryEnrichmentProviderBuilderExtensions to maintain the same word order as OpenTelemetryEnrichmentServiceCollectionExtensions